src/dextrader/nomics/pairs.py

```diff
@@ -0,0 +1,36 @@
+nomics = {"chain": {}}
+
```

Comment:
now that this is starting to gel for me, was curious if tomorrow you could walk through the process of discovering/finding the hash value for a particular chaing/exchange/market.